tests/e2e/utils.js

```diff
@@ -0,0 +1 @@
+export const waitForWpAdmin = () => page.waitForSelector( 'body.wp-admin' );
```

Comment:
Can you mark this utility behaviour in the e2e README, so that other contributors know it is available to use?